src/article/TextureEditingInterface.js

```diff
@@ -17,21 +17,4 @@ export default class TextureEditingInterface extends EditingInterface {
     return this.getDocument().createElement(...args)
   }
 
-  /*
-    2.0 API suggestion (pass only id, not data)
-  */
-  insertBlockNode(nodeId) {
```

Comment:
@michael I think we want to go for a purely Node/Element oriented interface. Using the `id` I consider more low-level and thus more 1.0 style.